onnxruntime/python/tools/transformers/models/llama/requirements-cuda.txt

```diff
@@ -1,4 +1,4 @@
 -r requirements.txt
-# Please manually install torch>=2.0.1 with CUDA enabled for the CUDA version installed in your system.
+# Please manually install torch>=2.2.0.dev20230920 with CUDA enabled for the CUDA version installed in your system.
```

Comment:
Is 2.1.0 enough (except dynamo export)?

Reply:
No, the nightly version is needed to fix [this issue](https://github.com/pytorch/pytorch/issues/109292) in the TorchScript exporter.

Reply:
Then we can add a check in the export script to ensure torch version >= 2.2.

Reply:
Added